### ISSUE TITLE
Trigot class broken

### ISSUE THREAD

**Issue by alerque** [sile-typesetter/sile#1225]
Overhauling the examples for the website I can't make the triglot one work. It doesn't look like it's the example, the class itself is broken.

```sil
\begin[class=triglot,papersize=a5]{document}
\end{document}
```

= crash

**Comment by Omikhleia** [comment]
On current master at this date, I'm not experience a crash with the above script, and the following also runs with a result that seems to be what's expected.

```
\begin[papersize=a5,class=triglot]{document}
\left LEFT
\middle MID
\right RIGHT
\sync
\end{document}
```

**Comment by alerque** [comment]
This got fixed in 71af1a94b when I overhauling all the classes. I'll re-enable the triglot example on the website. That being said the example there is exibiting other breakage with triglot+mirrored masters that I'll open a new issue for.
